packages/katello/katello/katello.spec

```diff
@@ -152,7 +152,6 @@ Requires: qpid-tools
 Requires: /bin/ps
 Requires: rh-mongodb34
 Requires: %{name}-service
```

Comment:
This should be changed to `foreman-maintain` since `katello-service` is no longer used.

Reply:
Can I do an atomic follow up for that change?

Reply:
Yes.